src/context/authStore.test.tsx

```diff
@@ -146,7 +146,7 @@ describe("AuthStoreContextProvider", () => {
     await waitFor(() => {
       expect(mockCaptureException).toHaveBeenCalledTimes(1);
     });
-  });
+  }, 60000);
```

Comment:
is this the timeout you set to fix the test 😆

Reply:
Yeah, this test and another test in `campaignConfigsStore.test.tsx`. Both of them are performing the same test that waits for `Sentry` to capture the exception 🤔

Reply:
I see I see! Hmm... does this part makes sense ah? Or should we mock that Sentry returns ok, since this test seems to be focused on testing other logic. What do you think? 🤔

Reply:
@sebastianquek what do you think? Good to remove the tests? :P

Reply:
The purpose of this test is to ensure that when there's a malformed config, `JSON.parse` will throw an error that should be handled appropriately. Testing the outcome that Sentry is triggered is one possible way. There are other possible ways such as checking the `ErrorBoundaryContent`, but it's out of scope for this story.

So given the 60s works, let's keep it this way and look closer into this issue separately?
Have added a card here: https://www.notion.so/sally-wallet/Investigate-if-timeout-of-60s-on-mobile-app-tests-can-be-removed-a322a3d719ce4a55aaf4aa110a0f7bfd

Reply:
I am okay with this, we can tackle this as a chore in the future!

Reply:
Sounds good~ thanks Seb! ^^